code/modules/clothing/spacesuits/rig.dm

```diff
@@ -473,6 +488,51 @@
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank,/obj/item/weapon/storage/firstaid,/obj/item/device/healthanalyzer,/obj/item/stack/medical,/obj/item/device/rad_laser)
 	armor = list(melee = 10, bullet = 5, laser = 10, energy = 5, bomb = 10, bio = 100, rad = 50)
 
+	//Research Director hardsuit
+/obj/item/clothing/head/helmet/space/rig/rd
+	name = "prototype hardsuit helmet"
+	desc = "A prototype helmet designed for research in a hazardous, low pressure environment. Scientific data flashes across the visor."
+	icon_state = "rig0-rd"
+	item_color = "rd"
+	burn_state = FIRE_PROOF
+	unacidable = 1
+	var/onboard_hud_enabled = 0 //stops conflicts with another diag HUD
+	max_heat_protection_temperature = FIRE_SUIT_MAX_TEMP_PROTECT
+	armor = list(melee = 30, bullet = 5, laser = 10, energy = 5, bomb = 100, bio = 100, rad = 60)

```

Comment:
lower the melee to be in line with all our other hardsuit's melee armor; we never ported over the "buff to all hardsuit melee stats"; this shouldn't get it either.